src/Symfony/Component/Console/ConsoleEvents.php

```diff
@@ -27,6 +27,14 @@ final class ConsoleEvents
      */
     const COMMAND = 'console.command';
 
+    /**
+     * The SIGNAL event allows you to perform some actions
+     * after the command execution was interrupted.
+     *
+     * @Event("Symfony\Component\Console\Event\ConsoleSignalEvent")
+     */
+    const SIGNAL = 'console.signal';
```

Comment:
This constant should be inserted into this collection:

https://github.com/symfony/symfony/blob/3b38f38add092b5a3faced331b4ba0c246fc18f5/src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml#L9-L33

Alternatively: Don't introduce the constant and use the event's FQCN as event name.

Reply:
Thank you. Added to the list